src/components/QuestDetails/components/StreakProgress/index.tsx

```diff
@@ -58,6 +58,7 @@ function PlayStreakBolts({
 }
 
 export default function StreakProgress({
+  standby,
```

Comment:
Unfortunately, I couldn't prevent adding this optional prop to the streak progress. Currently, there is no way to show this version of the content. We need it because on the quest page, the play streak percentage shouldn't be increased.

<img width="1009" alt="image" src="https://github.com/user-attachments/assets/f86c1df0-a3cf-4dd5-9fbc-45f28c447825">

Reply:
this is fine, but I think we could also use this prop to prevent the update function from being called. Then the user could see their progress on the quest page in the client

Reply:
This is a great call; with that, this prop is not needed, thanks!